ui/client/components/graph/focusable.tsx

```diff
@@ -0,0 +1,18 @@
+import React, {forwardRef, useCallback, MouseEventHandler} from "react"
```

Comment:
What is it for? To be able to select sth on graph pane?

Reply:
to grab focus e.g. from comment input when clicked on graph